src/components/StyledInput.tsx

```diff
@@ -15,6 +15,7 @@ interface Props {
   label?: string;
   placeholder?: string;
   style?: HTMLAttributes<HTMLInputElement>;
+  styleText?: HTMLAttributes<HTMLInputElement>;
```

Comment:
textStyle

Reply:
Yes, but I can't add two different styles to one parameter, when I wanted to add style for input and additional one for text from typography.ts.

Reply:
```
 className={css(
            style,
            styleText,          
)}
```
u spread  it in the same place

Reply:
In first time I was trying to set two styles to only one parameter, but then I saw error or it didn't works. Is it not corect when we want sometimes change only input style without our typography? You know, when I set one style for our component in place where we use them, I can't there add  our typography for typography.ts into aphrodite content and pass it in parameter.